### PR TITLE
Update contextBroker.spec

### PR DESCRIPTION
**Pull request by Anjali-NEC**
Fix issue https://github.com/telefonicaid/fiware-orion/issues/712

### CODE REVIEW

File: rpm/SPECS/contextBroker.spec [hunk 1]
```diff
@@ -72,7 +72,7 @@ fi
 
 %pre
 getent group %{owner} >/dev/null || groupadd -r %{owner}
-getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /opt/orion -s /bin/bash -c 'Orion account' %{owner}
+getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /usr/share/contextBroker/ -s /bin/bash -c 'Orion account' %{owner}
```
Comment (fgalan) [inline rpm/SPECS/contextBroker.spec:75]: Better to use:

```suggestion
getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /home/orion/ -s /bin/bash -c 'Orion account' %{owner}
```

CC: @cesarjorgemartinez

Comment (Anjali-NEC) [reply rpm/SPECS/contextBroker.spec]: Fixed in https://github.com/telefonicaid/fiware-orion/pull/3863/commits/8ba3fec7bdd26793b70ad14d9ffcd6bcc33520af

File: rpm/SPECS/contextBroker.spec [hunk 2]
```diff
@@ -72,7 +72,7 @@ fi
 
 %pre
 getent group %{owner} >/dev/null || groupadd -r %{owner}
-getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /opt/orion -s /bin/bash -c 'Orion account' %{owner}
+getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /home/orion/ -s /bin/bash -c 'Orion account' %{owner}
```
Comment (fgalan) [inline rpm/SPECS/contextBroker.spec:75]: An additional improvement, pls:

```suggestion
getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /home/orion -s /bin/bash -c 'Orion account' %{owner}
```

Comment (Anjali-NEC) [reply rpm/SPECS/contextBroker.spec]: Fixed in https://github.com/telefonicaid/fiware-orion/pull/3863/commits/c784068e40923126d867fbc253c8263abfca960f

**Review by fgalan — APPROVED** [review]
LGTM. Thanks!

**Comment by cesarjorgemartinez** [comment]
LGTM

**Pull request merged**